x-pack/plugins/alerting/server/task_runner/fixtures.ts

```diff
@@ -243,7 +243,7 @@ export const generateEventLog = ({
           ...((!isNil(numberOfTriggeredActions) || !isNil(numberOfScheduledActions)) && {
             metrics: {
               number_of_triggered_actions: numberOfTriggeredActions,
-              number_of_scheduled_actions: numberOfScheduledActions,
+              number_of_generated_actions: numberOfScheduledActions,
```

Comment:
Should we rename the variable to `numberOfGeneratedActions` as well?

Reply:
Good idea. I initially thought it would cause backwards compatibility issues on the executionStore, but it turns out it's a runtime variable (not persisted) :phew: Done in commit https://github.com/elastic/kibana/pull/130315/commits/1c8794e7ee470e4c1333390f0817f487dfe8153e.